### ISSUE TITLE
Identity pose swapping

### ISSUE THREAD

**Issue by Draylar** [Draylar/identity#56]
Players should be able to switch through entity poses. For example, a player morphed as a Dog should be able to lie down and sit.

**Comment by realflow100** [comment]
EPIC idea! :+1: :1st_place_medal:

**Comment by realflow100** [comment]
I dont think this has been done before with a morph mod yet so this would be the first! and most epic awesome idea! :+1:

**Comment by James103** [comment]
Example for Armor Stand identities:
1. No arms
2. Show arms
3. Show arms and use humanoid animations

(the ability key could be used to toggle showing the base plate, with no need to sneak)
